tests/test_fieldset.py

```diff
@@ -492,6 +492,17 @@ def generate_dataset(xdim, ydim, zdim=1, tdim=1):
     assert np.allclose(pset[0].lon, 4.5) and np.allclose(pset[0].lat, 10)
 
 
+@pytest.mark.parametrize('mode', ['scipy', 'jit'])
+def test_fieldset_frompop(mode):
+    filenames = path.join(path.join(path.dirname(__file__), 'test_data'), 'POPtestdata_time.nc')
+    variables = {'U': 'U', 'V': 'V', 'W': 'W', 'T': 'T'}
```

Comment:
Could you just remove the `W` field?

Reply:
Well, whether or not there is a `W` field has nothing to do with 2D or 3D interpolation. That is controlled by the `depth` dimensions

Reply:
Yes, it's just that we usually don't put `W` field in 2D. But you are right that it works.
So basically we can choose before testing more (like it is: assessing that `W` does not affect anything) or having something more similar to a real application (that is also easier to read as an example).

Reply:
So yes, maybe better to keep it

Reply:
So merge?